transport_nantes/transport_nantes/settings_local.py.DEV.template

```diff
@@ -28,22 +29,19 @@ DATABASES = {
 LOG_DIR = '/tmp/django-tn-log/'
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 
 ## Not needed in development, but maybe need to be set.
 AWS_ACCESS_KEY_ID = ''
 AWS_SECRET_ACCESS_KEY = ''
 AWS_DEFAULT_REGION = ''
 DEFAULT_FROM_EMAIL = ''
-
-STRIPE_PUBLISHABLE_KEY = 'pk_test_...'
-STRIPE_SECRET_KEY = 'sk_test_...'
-STRIPE_ENDPOINT_SECRET = 'whsec_...'
-
-MEDIA_URL = '/media/'
-MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+AWS_CONFIGURATION_SET_NAME = ''
 
 # LOCAL AND BETA SETTINGS, SAFE TO PUBLISH
 STRIPE_PUBLISHABLE_KEY = "pk_test_gVh6nSOJh19KHLsPepK35JzE00CkdNB1nR"
 STRIPE_SECRET_KEY = "sk_test_JA5eW82rv6lp61vNAwRUFIrr00rhzXUsLW"
```

Comment:
As noted, this stripe key must now be IMMEDIATELY changed on the strip dashboard.